cni/network/network.go

```diff
@@ -1125,18 +1126,23 @@ func (plugin *NetPlugin) Delete(args *cniSkel.CmdArgs) error {
 		// network ID is passed in and used only for migration
 		// otherwise, in stateless, we don't need the network id for deletion
 		epInfos, err = plugin.nm.GetEndpointState(networkID, args.ContainerID)
+		// if stateless CNI fail to get the endpoint from CNS for any reason other than  Endpoint Not found
+		if err != nil && !strings.Contains(err.Error(), "endpoint state could not be found in the statefile") {
```

Comment:
Is there any way we can tie this back to the defined error message? 

https://github.com/Azure/azure-container-networking/blob/6b05df11f4fb6b899ead7a3689b4ef60cc4a0482/cns/restserver/ipam.go#L29

Reply:
Resolved in the async delete PR